src/java/org/apache/cassandra/cql3/restrictions/SingleRestriction.java

```diff
@@ -19,7 +19,7 @@
 
 import org.apache.cassandra.cql3.QueryOptions;
 import org.apache.cassandra.cql3.statements.Bound;
-import org.apache.cassandra.db.MultiCBuilder;
+import org.apache.cassandra.db.MultiClusteringBuilder;
```

Comment:
Nit: The JavaDoc of some methods on this class still has some mentions of `<code>MultiCBuilder</code>` that should be updated to `<code>MultiClusteringBuilder</code>`. Or, even better, `{@link MultiClusteringBuilder}`.

Reply:
Renamed everywhere and created links.